front/tsconfig.json

```
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
```

compilerOptionsに
```suggestion
  "compilerOptions": {
    "outdir": "./build"
```
って書くと、コンパイル（=`tsc`）したときに、jsファイルが指定したディレクトリ以下にできるから、今みたいに同じ場所にjsファイルができてしまってごちゃごちゃにならなくてすむよ

---

そんでjsファイルは全部消しといたほうがいいかも。
いらないし